docs/source/reporters.rst

```diff
@@ -251,4 +251,20 @@ Note that this makes it really easy for your password to be picked up by
 software running on your machine, by other users logged into the system
 and/or for the password to appear in log files accidentally.
 
+XMPP
+___
+
+You can have notifications sent to you through the `XMPP protocol`.
+
+To achieve this, you first need to register a XMPP account for the bot.
+
+Here is a sample configuration:
+
+.. code:: yaml
+
+   xmpp:
+     enabled: true
+     sender: "BOT_ACCOUNT_NAME"
+     recipient: "YOUR_ACCOUNT_NAME"
+     password: "BOT_ACCOUNT_PASSWORD"
```

Comment:
IMHO visually it make sense to group "sender" and "password", even if it doesn't make any difference functionality-wise. Maybe also add some information that storing the password in the configuration in plaintext is insecure and shouldn't be done with the primary XMPP/Jabber account but rather one specifically created for urlwatch.